Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/DictationInputController.cs

```diff
@@ -1,70 +1,44 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.Devices;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Managers;
 using Microsoft.MixedReality.Toolkit.Core.Utilities.Async;
 using Microsoft.MixedReality.Toolkit.Core.Utilities.Async.AwaitYieldInstructions;
+using System.Text;
 using System.Threading.Tasks;
 using UnityEngine;
 
 #if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
-using System.Text;
 using UnityEngine.Windows.Speech;
 #endif
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem.Sources
+namespace Microsoft.MixedReality.Toolkit.Core.Devices.VoiceInput
 {
     /// <summary>
-    /// Singleton class that implements the DictationRecognizer to convert the user's speech to text.
-    /// The DictationRecognizer exposes dictation functionality and supports registering and listening for hypothesis and phrase completed events.
+    /// A Windows Mixed Reality Controller Instance.
     /// </summary>
-    public class DictationInputSource : BaseGenericInputSource
+    public class DictationInputController : BaseController, IMixedRealityDictationController
     {
         /// <summary>
-        /// Is the Dictation Manager currently running?
+        /// The Current Input System for this Input Source.
         /// </summary>
-        public static bool IsListening { get; private set; } = false;
-
-#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+        public static new IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>());
+        private static IMixedRealityInputSystem inputSystem = null;
 
         /// <summary>
-        /// Constructor.
+        /// The Current Input Source for this controller.
         /// </summary>
-        public DictationInputSource() : base("Dictation")
-        {
-            if (!Application.isPlaying) { return; }
+        private static IMixedRealityInputSource inputSource = null;
```

Comment:
We can probably get rid of this as well. The base class has an input source reference